test/fixtures/dialects/exasol/CreateFunctionStatement.yml

```diff
@@ -57,7 +57,9 @@ file:
     - expression:
         column_reference:
           identifier: res
-        binary_operator: '||'
+        binary_operator:
+        - pipe: '|'
+        - pipe: '|'
```

Comment:
Don't particularly like this being two parts. Which is part of the reason I've suggested some of the reverts above.